server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java

```diff
@@ -542,7 +542,7 @@ public static boolean submitAuditCVR(final CountyDashboard the_cdb,
                        the_audit_cvr.id() + " for county " + the_cdb.id() + 
                        ", cvr " + the_cvr_under_audit.id());
     }
-
+    Persistence.flush();
```

Comment:
@dmz tells me that this `flush` is to ensure that a later `read` is guaranteed to see the earlier changes.